src/webextension/widgets/password-input.js

```diff
@@ -58,14 +57,10 @@ export default class PasswordInput extends React.Component {
                type={showPassword ? "text" : "password"}
                ref={(element) => this.inputElement = element} {...props}/>
         <ButtonStack selectedIndex={selectedIndex}>
-          <Localized id="password-input-show">
-            <button type="button" disabled={disabled}
-                    onClick={() => this.showPassword(true)}>sHOw</button>
-          </Localized>
-          <Localized id="password-input-hide">
-            <button type="button" disabled={disabled}
-                    onClick={() => this.showPassword(false)}>hIDe</button>
-          </Localized>
+          <button className={styles.showBtn} type="button" title="Show" disabled={disabled}
```

Comment:
The "Show" and "Hide" titles are now not localized which means users of other languages will see English. I think we will need @jimporter's help here as I've previously struggled getting params within elements properly localized.

Reply:
In the diff for `src/webextension/locales/en-US/widgets.ftl` where localized strings are removed, you can see examples of how to tell Fluent to localize a prop (i.e. something like `title="Show"` in the diff above).

To be nice to future engineers, we (usually) also fake-localize the prop in the JS like `title="sHOw"`; this gets overridden immediately by Fluent, so it doesn't change the behavior, but it's a nice way of documenting what a component's localization should be like. We can also use the fake-localization in unit tests.

Reply:
OK thanks for pointer! I will take a quick look at getting this back to a localized state.